### PR TITLE
refactor: Remove any insensitive terms in codebase.

### PR DESCRIPTION
**Pull request by ren-tham**

### CODE REVIEW

File: projects/storefrontstyles/scss/_page-template.scss [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
-// TODO (#11607) remove LoginPageTemplate from whitelist
+// TODO (#11607) remove LoginPageTemplate from allowlist
 
-// The $page-template-blacklist can be used to prevent template CSS
+// The $page-template-blocklist can be used to prevent template CSS
 // being generated.
-$page-template-blacklist: () !default;
+$page-template-blocklist: () !default;
 
 // whitelisted page template selectors are processed
```
Comment (LTiger14) [inline projects/storefrontstyles/scss/_page-template.scss]: `whitelisted` in the comment

File: projects/storefrontstyles/scss/_page-template.scss [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
-// unless (some) they are blacklisted using `$page-template-blacklist`
-$page-template-whitelist: LandingPage2Template, ContentPage1Template,
+// unless (some) they are blacklisted using `$page-template-blocklist`
```
Comment (LTiger14) [inline projects/storefrontstyles/scss/_page-template.scss]: `blacklisted` in the comment

File: feature-libs/storefinder/styles/layout/_index.scss [hunk 3]
```diff
@@ -1,16 +1,16 @@
 @import './store-finder-page';
 
-// The blacklist can be used to prevent template CSS
+// The blocklist can be used to prevent template CSS
 // being generated.
-$page-template-blacklist-store-finder: () !default;
+$page-template-blocklist-store-finder: () !default;
 
 // whitelisted page template selectors are processed
```
Comment (LTiger14) [inline feature-libs/storefinder/styles/layout/_index.scss]: `whitelisted` is still in the comment

File: feature-libs/organization/administration/styles/_page-template.scss [hunk 4]
```diff
@@ -1,16 +1,16 @@
 @import 'company-page-template';
 
-// The blacklist can be used to prevent template CSS
+// The blocklist can be used to prevent template CSS
 // being generated.
-$page-template-blacklist-organization: () !default;
+$page-template-blocklist-organization: () !default;
 
 // whitelisted page template selectors are processed
```
Comment (LTiger14) [inline feature-libs/organization/administration/styles/_page-template.scss]: `whitelisted` is still in the comment.

File: feature-libs/organization/administration/styles/_page-template.scss [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
 // unless (some) they are blacklisted
```
Comment (LTiger14) [inline feature-libs/organization/administration/styles/_page-template.scss]: `blacklisted` is still in the comment.

**Review by LTiger14 — CHANGES_REQUESTED** [review]
I also noticed that the `mock-url.pipe.ts` and the `url-testing.module.ts` contain insensitive terms.

Since these files are used for testing it shouldn't be too hard to change those terms.

File: projects/storefrontstyles/scss/_page-template.scss [hunk 6]
```diff
@@ -2,23 +2,23 @@
...
+// The $page-template-blocklist can be used to prevent template CSS
 // being generated.
-$page-template-blacklist: () !default;
+$page-template-blocklist: () !default;
 
-// whitelisted page template selectors are processed
-// unless (some) they are blacklisted using `$page-template-blacklist`
-$page-template-whitelist: LandingPage2Template, ContentPage1Template,
+// allowlisted page template selectors are processed
+// unless (some) they are blocklisted using `$page-template-blocklist`
```
Comment (LTiger14) [inline projects/storefrontstyles/scss/_page-template.scss]: [nitpick] could you change this to `// unless (some) that are blocklisted using `$page-template-blocklist``

File: feature-libs/storefinder/styles/layout/_index.scss [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
+// The blocklist can be used to prevent template CSS
 // being generated.
-$page-template-blacklist-store-finder: () !default;
+$page-template-blocklist-store-finder: () !default;
 
-// whitelisted page template selectors are processed
-// unless (some) they are blacklisted
-$page-template-whitelist-store-finder: StoreFinderPageTemplate !default;
+// allowlisted page template selectors are processed
+// unless (some) they are blocklisted
```
Comment (LTiger14) [inline feature-libs/storefinder/styles/layout/_index.scss]: [nitpick] could you change this to `// unless (some) that are blocklisted using `$page-template-blocklist``

File: feature-libs/organization/administration/styles/_page-template.scss [hunk 8]
```diff
@@ -1,16 +1,16 @@
...
+// The blocklist can be used to prevent template CSS
 // being generated.
-$page-template-blacklist-organization: () !default;
+$page-template-blocklist-organization: () !default;
 
-// whitelisted page template selectors are processed
-// unless (some) they are blacklisted
-$page-template-whitelist-organization: CompanyPageTemplate !default;
+// allowlisted page template selectors are processed
+// unless (some) they are blocklisted
```
Comment (LTiger14) [inline feature-libs/organization/administration/styles/_page-template.scss]: [nitpick] could you change this to `// unless (some) that are blocklisted using `$page-template-blocklist``

**Review by LTiger14 — CHANGES_REQUESTED** [review]

**Review by LTiger14 — CHANGES_REQUESTED** [review]
I forgot to mention could you add your changes to the deprecation documentation.

**Review by LTiger14 — DISMISSED** [review]

**Review by marlass — APPROVED** [review]

**Review by Platonn — APPROVED** [review]

**Pull request merged**